code/modules/admin/random.dm

```diff
@@ -0,0 +1,44 @@
+#define ON_PURRBATION(H) (!(H.dna.features["tail_human"] == "None" && H.dna.features["ears"] == "None"))
+
+/proc/mass_purrbation()
```

Comment:
This can all just go in code\modules\admin\verbs\randomverbs.dm, no real point in a new file like this.